integration

```diff
@@ -993,6 +993,7 @@
   "wlcrs/huawei_solar",
   "wolffshots/hass-audiobookshelf",
   "xannor/ha_reolink_discovery",
+  "xiaodong-lx/run-command",
```

Comment:
Readme does not tell me anything about what this actually are for.
From the code it looks like a copy of the command_line integration.